Cargo.toml

```
@@ -176,6 +176,7 @@ memory_addresses = { version = "0.2.3", default-features = false, features = [
 
 [target.'cfg(target_arch = "aarch64")'.dependencies]
 aarch64 = { version = "0.0.14", default-features = false }
+arm-pl011-uart = { version = "0.3", default-features = false }
 arm-gic = { version = "0.6" }
```

Let's keep this sorted:

```suggestion
arm-gic = { version = "0.6" }
arm-pl011-uart = { version = "0.3", default-features = false }
```